### ISSUE TITLE
Update LICENSE file

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3079]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#3079|
|Type|enhancement|
|Created|2008-03-16 15:49:10Z|
|Branch|https://github.com/twisted/twisted/tree/license-update-3079|

I want some Internet fame :). There were probably a few other contributors since last time it has been updated.

<details><summary>Searchable metadata</summary>

```
trac-id__3079 3079
type__enhancement enhancement
reporter__therve therve
priority__normal normal
milestone__ 
branch__branches_license_update_3079 branches/license-update-3079
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1205682550000000 1205682550000000
changetime__1207514327000000 1207514327000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve commented|
|-|-|

(In [23081]) Branching to 'license-update-3079'

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix set owner to @therve|
|-|-|

+1 please merge

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set status to `closed`|
|-|-|

(In [23083]) Merge license-update-3079

Author: therve
Reviewer: radix
Fixes [#3079](https://github.com/twisted/twisted/issues/3079)

Update LICENSE file: update year to 2008 and add some contributors of the past
year.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve commented|
|-|-|

(In [23083]) Merge license-update-3079

Author: therve
Reviewer: radix
Fixes [#3079](https://github.com/twisted/twisted/issues/3079)

Update LICENSE file: update year to 2008 and add some contributors of the past
year.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_6"></a>|@therve commented|
|-|-|

(In [23083]) Merge license-update-3079

Author: therve
Reviewer: radix
Fixes [#3079](https://github.com/twisted/twisted/issues/3079)

Update LICENSE file: update year to 2008 and add some contributors of the past
year.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
